drivers/driver_functions.go

```diff
@@ -54,17 +35,17 @@ func ScaleService(payload map[string]interface{}) (int, map[string]interface{},
 	serviceID := payload["serviceId"].(string)
 	scaleAction := payload["scaleAction"].(string)
 	scaleChange := payload["scaleChange"].(float64)
-	url := fmt.Sprintf("%s/accounts/%s", CattleURL, projectID)
+	url := fmt.Sprintf("%s/accounts/%s", envConfig.CattleURL, projectID)
 
 	apiClient, err := client.NewRancherClient(&client.ClientOpts{
```

Comment:
Per my comment about adding rancher client as a parameter to the Execute interface, You'll have to pul this logic out to the framework.